src/nstime.c

```diff
@@ -152,12 +152,41 @@ nstime_monotonic_impl(void) {
 }
 nstime_monotonic_t *JET_MUTABLE nstime_monotonic = nstime_monotonic_impl;
 
+static void
+nstime_get_realtime(nstime_t *time) {
+  struct timespec ts;
```

Comment:
Nit -- we use freebsd kernel style, and tabs for indentation.

Reply:
> Just a couple of style nits now. If you could squash down to 1 commit, I think this is good to go.

I had a few problems with my squash but I hopefully everything is right now. Apologies up front if I missed anything.